src/services/refactors/generateGetAccessorAndSetAccessor.ts

```diff
@@ -214,11 +214,17 @@ namespace ts.refactor.generateGetAccessorAndSetAccessor {
     }
 
     function insertAccessor(changeTracker: textChanges.ChangeTracker, file: SourceFile, accessor: AccessorDeclaration, declaration: AcceptedDeclaration, container: ContainerDeclaration) {
-        isParameterPropertyDeclaration(declaration)
-            ? changeTracker.insertNodeAtClassStart(file, <ClassLikeDeclaration>container, accessor)
-            : isPropertyAssignment(declaration)
-                ? changeTracker.insertNodeAfterComma(file, declaration, accessor)
-                : changeTracker.insertNodeAfter(file, declaration, accessor);
+        if (isParameterPropertyDeclaration(declaration)) {

```

Comment:
@uniqueiniquity It was changed in [this](https://github.com/microsoft/TypeScript/pull/31777/commits/5902b327db2bd7bac6f5140706bab15146e8d7fa) commit - `no-unused-expressions`. `no-unused-expressions` throws an error for this part of code. 

There were two ways how to fix it, 

1. Add option `allowTernary` to `no-unused-expressions`
2. Change Ternary to `If/Else` statement

I decided to leave no-unused-expressions more `strict` because only one function required changes, and `TSLint::no-unused-expressions` was used without an additional option for `Ternary`. If you prefer the first way, just confirm it and I'll add  `allowTernary` option to `no-unused-expressions`.

Reply:
Yes, let's do option 1 - add allowTernary.

Reply:
@uniqueiniquity changed